ext/intl/transliterator/transliterator_class.c

```diff
@@ -173,7 +163,10 @@ static zend_object *Transliterator_clone_obj( zend_object *object )
 			err_msg = intl_error_get_message( TRANSLITERATOR_ERROR_P( to_orig ) );
 			zend_throw_error( NULL, "%s", ZSTR_VAL(err_msg) );
 			zend_string_free( err_msg ); /* if it's changed into a warning */
-			/* do not destroy tempz; we need to return something */
+		}
+		else
+		{
```

Comment:
```suggestion
		} else {
```
Style nit, I know intl is very inconsitent but let's not make it worse

Reply:
updated thanks